near-contract-standards/src/fungible_token/metadata.rs

```diff
@@ -0,0 +1,15 @@
+use near_sdk::serde::Serialize;
+
+#[derive(Serialize)]
+#[serde(crate = "near_sdk::serde")]
+pub struct FungibleTokenMetadata {
+    pub version: String,
+    pub name: String,
+    pub symbol: String,
+    pub reference: String,
```

Comment:
actually, I do think we want to add `reference_hash` as sha256 of `reference`. I do believe this makes sense, like:

    pub reference_hash: [u8; 32],

We have this in the Nomicon NEP here, too:
https://nomicon-pr-158.onrender.com/Standards/Tokens/FungibleTokenMetadata.html#reference-level-explanation

Reply:
Switched to base58 hash